File: frontend/scripts_migracion/extract_tabs.js
const fs = require('fs');
const glob = require('glob');
const path = require('path');
const files = glob.sync('c:/GD-rsp/APP-CuadernoFP/frontend/src/app/**/page.tsx');
files.forEach(f => {
  const content = fs.readFileSync(f, 'utf8');
  if (content.includes('TABS = [')) {
    const tabsMatch = content.match(/TABS\s*=\s*\[(.*?)\];/s);
    if (tabsMatch) {
      console.log('--- ' + path.basename(path.dirname(f)));
      const simplified = tabsMatch[1].replace(/<[^>]+>/g, '').replace(/className=\".*?\"/g, '');
      console.log(simplified.substring(0, 500));
    }
  }
});

File: scratch/extract_tabs.py
import os, re
for r, d, files in os.walk('frontend/src/app'):
    for f in files:
        if f == 'page.tsx':
            path = os.path.join(r, f)
            with open(path, encoding='utf8') as f_in:
                content = f_in.read()
            match = re.search(r'const TABS\s*=\s*\[(.*?)\];', content, re.DOTALL)
            if match:
                labels = re.findall(r'label:\s*(?:t\()?[\'\"\`]([^\'\"\`]+)[\'\"\`]', match.group(1))
                if labels:
                    print(f"{os.path.basename(r)}: {labels}")
